README.md

```diff
@@ -65,7 +65,7 @@ AMP is an open source project, and we'd love your help making it better!
 
 ## Other useful information
 
-- [AMP's release documentation](contributing/release-schedule.md) provides details on how and when AMP releases new versions, including [how to know when a change is live in production](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md#determining-if-your-change-is-in-production).
+- [AMP's release documentation](contributing/release-schedule.md) provides details on how and when AMP releases new versions, including [how to know when a change is live in production](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md#determining-if-your-change-is-ina release).
```

Comment:
This link worked when I copied and pasted it, but the space seems problematic to me.